refinery/data_set_manager/models.py

```diff
@@ -6,20 +6,21 @@
 from datetime import datetime
 import logging
 
-import requests
-from celery.result import AsyncResult
-from requests.exceptions import HTTPError
-
 from django.conf import settings
 from django.db import models
 from django_extensions.db.fields import UUIDField
 
-import core
-import file_store
-import data_set_manager
 
+from celery.result import AsyncResult
+import requests
+from requests.exceptions import HTTPError
+
+from .genomes import map_species_id_to_default_genome_build
+import data_set_manager
```

Comment:
I could not replace with relative imports for  data_set_manager.tasks.generate_auxiliary_file due to a circular import which I could not resolve.